docker/bootstrap/Dockerfile

```diff
@@ -28,7 +28,8 @@ RUN apt-get update && \
     libgtk-3-0
 
 COPY --chown=${USER_NAME} . ${OPENVINO_WORKBENCH_ROOT}
-RUN chmod +x -R ${OPENVINO_WORKBENCH_ROOT}/automation/bootstrap/
+RUN chmod +x -R ${OPENVINO_WORKBENCH_ROOT}/automation/bootstrap/ && \
```

Comment:
What if we make the files executable in the repo?

Reply:
In the repository they are already with the launch parameter. For some reason, it is deleted after building the image

Reply:
merge with master